corehq/apps/es/migration_operations.py

```diff
@@ -170,24 +176,63 @@ def __init__(self, name, type_, properties, comment=None):
             ``mapping._meta.comment`` property. If ``None`` (the default), then
             the existing ``mapping._meta.comment`` value (if one exists) is
             retained.
+        :param print_diff: Optional (default ``True``) set to ``False`` to
```

Comment:
It does make it more verbose. It also replicates the default behavior of the manage command it replaces (`update_es_mapping`). That being said, the "existing default behavior" that this replicates is relatively new, and was only used in production once (in other words, it's not a long-standing precedent). This is a bit tricky because exactly "who wants to see this diff" probably depends on who's running `./manage.py migration`, rather than "who is creating the migration". I would be happy to offline about these details.

Reply:
Fine to leave as is if you think it's valuable output for the person running `migrate`.